tests/test_ERC721_Metadata.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+import asyncio
+from starkware.starknet.testing.starknet import Starknet
+from utils import Signer, str_to_felt, assert_revert
+
+
+signer = Signer(123456789987654321)
+
+first_token_id = (5042, 0)
```

Comment:
I insist we need to clarify these are random values to avoid confusion

Reply:
Okay, on it!